app/cdap/components/WrangleHome/Components/WrangleCard/iconStore/BigQuerySVG.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+
+export const BigQuery = (
```

Comment:
Can these SVGs be removed?

Reply:
yes, these svgs will be removed.
but it will be part of another PR after #599